File: pgm/ABC_Java/ABC_463_A.java
import java.io.*;
// import java.math.*;
// import java.time.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final String[] arr = reader.readLine().split (" ");
        reader.close();
        final String Yes = "Yes";
        final String No = "No";
        final int one = 1;
        final int nine = 9;
        final int sixteen = 16;
        final int zero = 0;
        final int X = Integer.parseInt (arr[zero]);
        final int Y = Integer.parseInt (arr[one]);
        System.out.println (nine * X == sixteen * Y ? Yes : No);
        System.exit (0);
    }
}

File: pgm/ABC_C#/ABC_463_A.cs
using System;
// using System.Collections.Generic;
// using System.Globalization;
// using System.IO;
// using System.Linq;
// using System.Text;
// using System.Text.RegularExpressions

class Program
{
    static void Main()
    {
        string[] arr = Console.ReadLine().Split();
        const string Yes = "Yes";
        const string No = "No";
        const int one = 1;
        const int nine = 9;
        const int sixteen = 16;
        const int zero = 0;
        int X = int.Parse(arr[zero]);
        int Y = int.Parse(arr[one]);
        Console.WriteLine(nine * X == sixteen * Y ? Yes : No);
        Environment.Exit(0);
    }
}